Add First Steps topic (#7512)
DevExpress/devextreme-documentation: concepts/Common/First Steps/30 Explore More.md

To learn more about DevExtreme, refer to the following sources:

---
##### jQuery

1. Learn the basics with [Component Configuration Syntax](/Documentation/Guide/jQuery_Components/Component_Configuration_Syntax/).

2. Edit and copy code from over 450 [DevExtreme demos](https://js.devexpress.com/Demos/WidgetsGallery).

> 3. To learn more about a specific component, review guides and API reference in documentation. For common issues, refer to:

Reviewer: ```suggestion
3. To learn more about a specific component, review guides and API references in the documentation. For common issues, refer to the following resources:
```

    - [Troubleshooting Guide](/concepts/80%20Troubleshooting/00%20Basics '/Documentation/Guide/Troubleshooting/Basics/')
    - [Support Center Tickets](https://supportcenter.devexpress.com/ticket/list?preset=ab3eb0eb-09d3-42b5-ae17-53bed1e6cec3)
    - [Examples](https://supportcenter.devexpress.com/ticket/list?preset=c2515c12-d6f9-4dd5-8381-fa9978072519)

4. Need help? [Submit a ticket](https://supportcenter.devexpress.com/ticket/create) to the DevExpress Support Center.

##### Angular

1. Learn the basics with [Component Configuration Syntax](/Documentation/Guide/Angular_Components/Component_Configuration_Syntax/).

2. Edit and copy code from over 450 [DevExtreme demos](https://js.devexpress.com/Demos/WidgetsGallery).

> 3. To learn more about a specific component, review guides and API reference in documentation. For common issues, refer to:

Reviewer: ```suggestion
3. To learn more about a specific component, review guides and API references in the documentation. For common issues, refer to the following resources:
```

    - [Troubleshooting Guide](/concepts/80%20Troubleshooting/00%20Basics '/Documentation/Guide/Troubleshooting/Basics/')
    - [Support Center Tickets](https://supportcenter.devexpress.com/ticket/list?preset=ab3eb0eb-09d3-42b5-ae17-53bed1e6cec3)
    - [Examples](https://supportcenter.devexpress.com/ticket/list?preset=c2515c12-d6f9-4dd5-8381-fa9978072519)

4. Need help? [Submit a ticket](https://supportcenter.devexpress.com/ticket/create) to the DevExpress Support Center.

##### Vue

1. Learn the basics with [Component Configuration Syntax](/Documentation/Guide/Vue_Components/Component_Configuration_Syntax/).

2. Edit and copy code from over 450 [DevExtreme demos](https://js.devexpress.com/Demos/WidgetsGallery).

> 3. To learn more about a specific component, review guides and API reference in documentation. For common issues, refer to:

Reviewer: ```suggestion
3. To learn more about a specific component, review guides and API references in the documentation. For common issues, refer to the following resources:
```

    - [Troubleshooting Guide](/concepts/80%20Troubleshooting/00%20Basics '/Documentation/Guide/Troubleshooting/Basics/')
    - [Support Center Tickets](https://supportcenter.devexpress.com/ticket/list?preset=ab3eb0eb-09d3-42b5-ae17-53bed1e6cec3)
    - [Examples](https://supportcenter.devexpress.com/ticket/list?preset=c2515c12-d6f9-4dd5-8381-fa9978072519)

4. Need help? [Submit a ticket](https://supportcenter.devexpress.com/ticket/create) to the DevExpress Support Center.

##### React

> 1. Learn the basics with [Component Configuration Syntax](/Documentation/Guide/React_Components/Component_Configuration_Syntax/).

Reviewer: ```suggestion
1. Learn the basics of [Component Configuration Syntax](/Documentation/Guide/React_Components/Component_Configuration_Syntax/).
```

2. Edit and copy code from over 450 [DevExtreme demos](https://js.devexpress.com/Demos/WidgetsGallery).

> 3. To learn more about a specific component, review guides and API reference in documentation. For common issues, refer to:

Reviewer: ```suggestion
3. To learn more about a specific component, review guides and API references in the documentation. For common issues, refer to the following resources:
```

To learn more about DevExtreme, refer to the following sources:

---
##### jQuery

1. Learn the basics with [Component Configuration Syntax](/Documentation/Guide/jQuery_Components/Component_Configuration_Syntax/).

2. Edit and copy code from over 450 [DevExtreme demos](https://js.devexpress.com/Demos/WidgetsGallery).

> 3. After understanding components, use individual component documentation for detailed exploration. For common issues, refer to:

Reviewer 2: It sounds strange that I need to use documentation after I already understood the component. 

```suggestion
3. To learn more about a specific component, review guides and API reference in documentation. For common issues, refer to:
```

    - [Troubleshooting Guide](/concepts/80%20Troubleshooting/00%20Basics '/Documentation/Guide/Troubleshooting/Basics/')
    - [Support Center Tickets](https://supportcenter.devexpress.com/ticket/list?preset=ab3eb0eb-09d3-42b5-ae17-53bed1e6cec3)
    - [Examples](https://supportcenter.devexpress.com/ticket/list?preset=c2515c12-d6f9-4dd5-8381-fa9978072519)

> 4. Need further help? [Submit a ticket](https://supportcenter.devexpress.com/ticket/create) in the Support Center.

Reviewer 2: ```suggestion
4. Need help? [Submit a ticket](https://supportcenter.devexpress.com/ticket/create) to the DevExpress Support Center.
```

##### Angular

1. Learn the basics with [Component Configuration Syntax](/Documentation/Guide/Angular_Components/Component_Configuration_Syntax/).

2. Edit and copy code from over 450 [DevExtreme demos](https://js.devexpress.com/Demos/WidgetsGallery).

> 3. After understanding components, use individual component documentation for detailed exploration. For common issues, refer to:

Reviewer 2: ```suggestion
3. To learn more about a specific component, review guides and API reference in documentation. For common issues, refer to:
```

    - [Troubleshooting Guide](/concepts/80%20Troubleshooting/00%20Basics '/Documentation/Guide/Troubleshooting/Basics/')
    - [Support Center Tickets](https://supportcenter.devexpress.com/ticket/list?preset=ab3eb0eb-09d3-42b5-ae17-53bed1e6cec3)
    - [Examples](https://supportcenter.devexpress.com/ticket/list?preset=c2515c12-d6f9-4dd5-8381-fa9978072519)

> 4. Need further help? [Submit a ticket](https://supportcenter.devexpress.com/ticket/create) in the Support Center.

Reviewer 2: ```suggestion
4. Need help? [Submit a ticket](https://supportcenter.devexpress.com/ticket/create) to the DevExpress Support Center.
```

##### Vue

1. Learn the basics with [Component Configuration Syntax](/Documentation/Guide/Vue_Components/Component_Configuration_Syntax/).

2. Edit and copy code from over 450 [DevExtreme demos](https://js.devexpress.com/Demos/WidgetsGallery).

> 3. After understanding components, use individual component documentation for detailed exploration. For common issues, refer to:

Reviewer 2: ```suggestion
3. To learn more about a specific component, review guides and API reference in documentation. For common issues, refer to:
```

    - [Troubleshooting Guide](/concepts/80%20Troubleshooting/00%20Basics '/Documentation/Guide/Troubleshooting/Basics/')
    - [Support Center Tickets](https://supportcenter.devexpress.com/ticket/list?preset=ab3eb0eb-09d3-42b5-ae17-53bed1e6cec3)
    - [Examples](https://supportcenter.devexpress.com/ticket/list?preset=c2515c12-d6f9-4dd5-8381-fa9978072519)

> 4. Need further help? [Submit a ticket](https://supportcenter.devexpress.com/ticket/create) in the Support Center.

Reviewer 2: ```suggestion
4. Need help? [Submit a ticket](https://supportcenter.devexpress.com/ticket/create) to the DevExpress Support Center.
```

##### React

1. Learn the basics with [Component Configuration Syntax](/Documentation/Guide/React_Components/Component_Configuration_Syntax/).

2. Edit and copy code from over 450 [DevExtreme demos](https://js.devexpress.com/Demos/WidgetsGallery).

> 3. Explore this website for more information about the components. 

Reviewer 2: ```suggestion
3. Explore this website for more information about DevExtreme components. 
```

4. For common issues, refer to:

    - [Troubleshooting Guide](/concepts/80%20Troubleshooting/00%20Basics '/Documentation/Guide/Troubleshooting/Basics/')
    - [Support Center Tickets](https://supportcenter.devexpress.com/ticket/list?preset=ab3eb0eb-09d3-42b5-ae17-53bed1e6cec3)
    - [Examples](https://supportcenter.devexpress.com/ticket/list?preset=c2515c12-d6f9-4dd5-8381-fa9978072519)

> 5. Need further help? [Submit a ticket](https://supportcenter.devexpress.com/ticket/create) in the Support Center.

Reviewer 2: ```suggestion
5. Need help? [Submit a ticket](https://supportcenter.devexpress.com/ticket/create) to the DevExpress Support Center.
```
